analysis/Scripts/MassFitting/Archive/CombinedBinFitting.py

```diff
@@ -1,28 +0,0 @@
-import ROOT, os, MassfitLib as mf 
-from CombinedBinFit_Dict import fittingDict
-
-#STOOMBOOT 
-#path = "/dcache/bfys/scalo/binned_files/"
-path = "/home/exultimathule/Code/HonoursProgramme/MassFitScript/testDirectories/"
-
-#STOOMBOOT 
-#PDFpath = "~/PDF_output/"
-#DictPath = "~/Dict_output/"
-PDFpath = "/home/exultimathule/Code/HonoursProgramme/MassFitScript/PDF_output/"
-DictPath = "/home/exultimathule/Code/HonoursProgramme/MassFitScript/Dict_output/"
-
-mainDict = {}
-years = [2011,2012,2015,2016,2017,2018]
-
-for year in years:
-	mainDict[year] = mf.combYPTbinShapeFit(year,"GaussCB", fittingDict, path, "both", PDFpath)
-
-# WRITES THE .py FILE WITH DICT AND dictSearch FUNCTION
```

Comment:
Any reason this is now deleted, give it was in the archive anyway?

Reply:
I basically thought about what we had said about having everything in older versions of the repo anyways, so I thought that it might just be cleaner to delete the massfitting archive. But see the answer to the cutmytuple